tests/components/alexa/test_smart_home.py

```diff
@@ -2378,7 +2378,7 @@ async def test_cover_position_range(
         "range": {"minimumValue": 1, "maximumValue": 100},
     } in position_state_mappings
 
-    call, msg = await assert_request_calls_service(
+    _, msg = await assert_request_calls_service(
```

Comment:
`call` might technically be unused. However, I would recommend against replacing it with `_`. There isn't really a performance benefit in doing so and it can help with debugging.

Personally, I suggest we disable that rule.

Reply:
Hmm, pylint is the same opinion as Ruff is and puts it as "Unused variable"